cli/test-output-directory.md

```diff
@@ -7,7 +7,7 @@ Maestro stores test artifacts such as screenshots, logs, and AI reports during t
 Use the `--test-output-dir` option to specify the output directory for a specific test run:
 
 ```bash
-maestro test workspace --test-output-dir=/your_maestro_output_directory
+maestro test workspace --test-output-dir=debug_output_directory
```

Comment:
If we use the term debug output directory here, is there any chance it'll get confused with the other one - https://docs.maestro.dev/troubleshooting/debug-output#maestro-logs ?